metals/src/main/scala/scala/meta/internal/builds/NewProjectProvider.scala

```diff
@@ -4,6 +4,7 @@ import java.io.File
 
 import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
+import scala.jdk.CollectionConverters._
```

Comment:
This was usually added via `MetalsEnrichments` and the `DecorateAsScala`/`DecorateAsJava` traits,  I think we can just switch to `AsJavaExtensions` and `AsScalaExtensions` instead.

Reply:
So I started out with that, but then I realized there was a lot of places that only brought in `MetalsEnrichments` just to use those instead of just using them directly. Do you still think we should do this?